### PR TITLE
Fixes comment for webhook conversion strategy

### PR DESCRIPTION
**Pull request by srm09**
This patch fixes the language of the comment for the webhook conversion strategy.

### CODE REVIEW

**Comment by satyampsoni** [comment]
@srm09 which issue is related to this PR?

File: content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md [hunk 1]
```diff
@@ -493,7 +493,7 @@ spec:
           port:
             type: string
   conversion:
-    # a Webhook strategy instruct API server to call an external webhook for any conversion between custom resources.
+    # the Webhook strategy instructs the API server to call an external webhook for any conversion between custom resources.
```
Comment (tengqm) [inline content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md:496]: This doesn't have to be "the webhook strategy", IMO.

Comment (sftim) [reply content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md:496]: - What are the conversions between? For example, if I have a FooBar named `thing1` and a different FooBar named `thing2`, is that a conversion (no)
   - it's a conversion between storage serializations, right?
- Optionally, mention what other options there are apart from `Webhook`

**Review by sftim — COMMENTED** [review]
/lgtm

If we merged this, it would slightly improve the docs compared to what we have.

**Comment by reylejano** [comment]
This PR improves the grammar in a few comment, it does not intend to change the message of the comment.
/approve

**Pull request merged**